doc/Tutorials/Exploring_Data.ipynb

```diff
@@ -432,7 +432,7 @@
      "cell_type": "markdown",
      "metadata": {},
      "source": [
-      "Now that we have removed the redundant Dimension we can create an NdLayout indexed just by the date:"
```

Comment:
I think "an NdLayout" is correct, not "a NdLayout", because we pronounce it en-dee-layout.  But that can be fixed afterwards.